mmv1/third_party/terraform/services/netapp/resource_netapp_active_directory_sweeper.go.erb

```diff
@@ -0,0 +1,131 @@
+<% autogen_exception -%>
+package netapp
+<% unless version == 'ga' -%>
```

Comment:
My only question is what's the reason for wrapping these in a beta wrapper when the resources are not beta? (they look to be GA)

This also allows us to mark these files as `.go` instead of `.go.erb`

Relevant Documentation that talks about version guards: https://googlecloudplatform.github.io/magic-modules/develop/promote-to-ga/#promote-fields-and-resources

Reply:
Done